test/Test.re

```diff
@@ -885,48 +1041,38 @@ let core = [
              </Div>
            ),
          )
-      |> executeSideEffects
-      |> expect(
-           ~label="IAmBox",
-           [
-             MountChild(div, box("ImABoxB"), 0),
-             UnmountChild(div, box("ImABoxA")),
-             MountChild(div, div, 1),
-           ],
-         )
-      |> ignore;
-    },
-  ),
-  (
-    "Test a <SingleChildDiv> with single Flat child, with a changing key",
-    `Quick,
-    () => {
+      |> executeSideEffects;
+    let mountLog = state |> getMountLogAndReset;
+
+    expect.list(mountLog).toEqual([
+      MountChild(div, box("ImABoxB"), 0),
+      UnmountChild(div, box("ImABoxA")),
+      MountChild(div, div, 1),
+    ]);
+  })
+});
+
+describe(
+  "Test a <SingleChildDiv> with single Flat child, with a changing key",
+  ({test}) => {
+  test("It re-mounts the node with the new key", ({expect}) => {
+    let key1 = Key.create();
+    let key2 = Key.create();
+    let state =
       render(
-        Components.(
-          <SingleChildDiv> ...<Div key={Key.create()} /> </SingleChildDiv>
-        ),
+        Components.(<SingleChildDiv> ...<Div key=key1 /> </SingleChildDiv>),
       )
       |> executeSideEffects
       |> reset
       |> update(
-           Components.(
-             <SingleChildDiv> ...<Div key={Key.create()} /> </SingleChildDiv>
-           ),
-         )
-      |> executeSideEffects
-      |> expect(
-           ~label="it re-mounts the node with the new key",
-           [
-             UnmountChild(singleChildDiv, div),
-             MountChild(singleChildDiv, div, 0),
-           ],
+           Components.(<SingleChildDiv> ...<Div key=key2 /> </SingleChildDiv>),
          )
-      |> ignore;
-    },
-  ),
-];
-
-/** Annoying dune progress */
-print_endline("");
+      |> executeSideEffects;
+    let mountLog = state |> getMountLogAndReset;
 
-Alcotest.run(~argv=[|"--verbose --color"|], "Brisk", [("Core", core)]);
```

Comment:
It is conserved ? `"Brisk", [("Core", core)]`

Reply:
I could nest the `describe` some more, do you want this @wokalski ?

Reply:
Not relevant anymore. It's a relict of the distant past.